app/test/std_sdk_test.go

```diff
@@ -29,6 +29,12 @@ import (
 	tmrand "github.com/tendermint/tendermint/libs/rand"
 )
 
+var (
+	oneUtia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
+	// oneTia is 1e6 utia (a.k.a one million utia)
+	oneTia  = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1e6)))
+)
```

Comment:
[optional, not blocking]

we could even put this in a test/util repo such as testfactory etc to make this usable by other tests

Reply:
yeah, have pushed the variables to a common file in test/util